.gitlab-ci.yml

```diff
@@ -299,6 +299,7 @@ publish-docs:
     # remove everything and restore generated docs, README and Jekyll config
     - rm -rf ./*
     - mv /tmp/doc/* .
+    - mv /tmp/doc/.images .
```

Comment:
What about doing `shopt -s dotglob` as suggested here https://askubuntu.com/a/259386/462538. This way it would copy all hidden files/folders.

Reply:
Yeah I thought about that too, but wasn't to sure about possible side effects, since we e.g. have a `.github` folder as well in the repo.